research/syntaxnet/Dockerfile

```diff
@@ -55,20 +56,22 @@ RUN python -m pip install \
     && rm -rf /root/.cache/pip /tmp/pip*
 
 # Installs the latest version of Bazel.
-RUN wget --quiet https://github.com/bazelbuild/bazel/releases/download/0.4.3/bazel-0.4.3-installer-linux-x86_64.sh \
-    && chmod +x bazel-0.4.3-installer-linux-x86_64.sh \
-    && ./bazel-0.4.3-installer-linux-x86_64.sh \
-    && rm ./bazel-0.4.3-installer-linux-x86_64.sh
+RUN wget --quiet https://github.com/bazelbuild/bazel/releases/download/0.5.4/bazel-0.5.4-installer-linux-x86_64.sh \
+    && chmod +x bazel-0.5.4-installer-linux-x86_64.sh \
+    && ./bazel-0.5.4-installer-linux-x86_64.sh \
+    && rm ./bazel-0.5.4-installer-linux-x86_64.sh
 
 COPY WORKSPACE $SYNTAXNETDIR/syntaxnet/WORKSPACE
 COPY tools/bazel.rc $SYNTAXNETDIR/syntaxnet/tools/bazel.rc
 COPY tensorflow $SYNTAXNETDIR/syntaxnet/tensorflow
 
+# Workaround solving the PYTHON_BIN_PATH not found problem
+ENV PYTHON_BIN_PATH=/usr/bin/python
 # Compile common TensorFlow targets, which don't depend on DRAGNN / SyntaxNet
 # source. This makes it more convenient to re-compile DRAGNN / SyntaxNet for
 # development (though not as convenient as the docker-devel scripts).
 RUN cd $SYNTAXNETDIR/syntaxnet/tensorflow \
-    && tensorflow/tools/ci_build/builds/configured CPU \
+    && ./configure CPU \
```

Comment:
This seems to be a typo?
`./configure` does not really take "CPU" as a command-line argument.

Reply:
That's true. I can do another PR if you wish to fix that. It's working properly with it but it is kind of misleading the user.